editor/inspector/editor_inspector.cpp

```diff
@@ -5117,6 +5117,24 @@ void EditorInspector::_edit_set(const String &p_name, const Variant &p_value, bo
 }
 
 void EditorInspector::_property_changed(const String &p_path, const Variant &p_value, const String &p_name, bool p_changing, bool p_update_all) {
+#ifdef TOOLS_ENABLED
+	// validate before edit value
+	String p_error_message;
+	bool is_valid = object->is_valid_property_value(p_path, p_value, p_error_message);
+	if (editor_property_map.has(p_path)) {
+		for (EditorProperty *E : editor_property_map[p_path]) {
+			E->set_draw_warning(!is_valid);
```

Comment:
```suggestion
	if (editor_property_map.has(p_path)) {
	        bool is_valid = object->is_valid_property_value(p_path, p_value, p_error_message);
		for (EditorProperty *E : editor_property_map[p_path]) {
			E->set_draw_warning(!is_valid);
```

No need to call `is_valid_property_value()` if the path doesn't exist in `editor_property_map`.